src/tests/ftest/container/query_attribute.py

```diff
@@ -7,6 +7,31 @@
 from apricot import TestWithServers
 
 
+# Test container set-attr, get-attr, and list-attrs with different
+# types of characters.
+test_strings = [
```

Comment:
I think this should disable the pylint errors for the backslashes
```suggestion
# pylint: disable=anomalous-backslash-in-string
test_strings = [
```

Reply:
I'll try that, thanks.